pkg/client/client.go

```diff
@@ -94,7 +98,7 @@ func (n *NutanixClientHelper) GetClientFromEnvironment(ctx context.Context, nuta
 	// Add env provider for CAPX manager
 	npe, err := n.getManagerNutanixPrismEndpoint()
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to create prism endpoint %w", err)
```

Comment:
nit: Can we get `:` between new contextual error string and wrapped error?
```suggestion
		return nil, fmt.Errorf("failed to create prism endpoint: %w", err)
```

Reply:
Done.